modules/run-consul/run-consul

```diff
@@ -478,7 +552,8 @@ function run {
       assert_not_empty "--cert-file-path" "$cert_file_path"
       assert_not_empty "--key_file_path" "$key_file_path"
     fi
-    generate_consul_config "$server" "$config_dir" "$user" "$cluster_tag_key" "$cluster_tag_value" "$datacenter" "$enable_gossip_encryption" "$gossip_encryption_key" "$enable_rpc_encryption" "$ca_path" "$cert_file_path" "$key_file_path"
+
+    generate_consul_config "$server" "$config_dir" "$user" "$cluster_tag_key" "$cluster_tag_value" "$datacenter" "$enable_gossip_encryption" "$gossip_encryption_key" "$enable_rpc_encryption" "$ca_path" "$cert_file_path" "$key_file_path" "$cleanup_dead_servers" "$last_contact_threshold" "$max_trailing_logs" "$server_stabilization_time" "$redundancy_zone_tag" "$disable_upgrade_migration" "$upgrade_version_tag"
```

Comment:
When your args list gets long, sometimes it's more readable if you use one arg per line. For example:

```bash
generate_consul_config \
  "$server" \
  "$config_dir" \
  "$user" \
   ...
```